far/keys.hpp

```diff
@@ -1292,6 +1292,12 @@ enum far_key_code
 	KEY_RALT_BASE            =KEY_RALT,
 	KEY_ALTSHIFT_BASE        =KEY_ALTSHIFT,
 	KEY_RALTSHIFT_BASE       =KEY_RALT|KEY_SHIFT,
+
+	KEY_ALTNUMPAD4           =KEY_ALT|KEY_NUMPAD4,
```

Comment:
This won't work  - Alt+Numpad keys are reserved for character code input.

Reply:
My mistake, removed